static-analysis/teammates-macker.xml

```diff
@@ -476,6 +476,21 @@
                     </deny>
                 </access-rule>
             </ruleset>
+            <ruleset name="java.util.Date/java.util.Calendar should not be used">
```

Comment:
Ok to do this now because there are still exceptions to the rule. When this becomes a code base-wide rule later on, use CheckStyle's `IllegalImport` rule instead.